terraform/modules/credentials/outputs.tf

```diff
@@ -1,4 +1,4 @@
 output "secrets" {
   sensitive = true
   value = local.low_pass_values
```

Comment:
Just checked in staging and the `directdebit-connector-secret-service` doesn't have the values from `pay-dev-pass`. I think it's because this output needs changing to merge in the `local.dev_pass_values` else they won't be available; this is only returning the `low_pass_values`